### ISSUE TITLE
Invalid OAuth parameter: client_id

### ISSUE THREAD

**Issue by isabelrios** [mozilla-lockbox/lockbox-ios#587]
Latest master:  7047fac
iOS 11.4 iPhone 6s
Not sure if there is still development pending or if this is already tracked. While doing some testing to fix the XCUITests, I get to this screen when I logout from the FxAccount Management page (where I'm redirect when disconnecting)

![captura de pantalla 2018-07-31 a las 15 44 52](https://user-images.githubusercontent.com/1897507/43463505-028c2314-94d9-11e8-89cb-26b1f65d73f8.png)

I have to do that in order to leave the FxAccount Management page (or is there another way?) This might be another bug, btw.

Depends on #588
Depends on #422

**Comment by devinreams** [comment]
Thanks, @isabelrios.

Technically the user shouldn't reach nor see this screen. I'll mark this as dependent on #588 and #422. Once those are fixed this can be safely closed.
